src/index.js

```diff
@@ -304,3 +310,7 @@ module.exports = class Bitwap {
     this.engine.stop()
   }
 }
+
+module.exports = Bitswap
+
+function noop () {}
```

Comment:
I might be wrong, but I don't see this as a style preference. Seeing `cb = cb || noop` is more simpler than `cb = cb || (() => {})` everytime I want a noop.

I can revert back though.

As I said at the top of the PR: "Nothing to see here", which was meant to save time from CR, this is far from complete (note that I didn't add the "needs review" label).